system_tests/common_test.go

```diff
@@ -263,6 +263,7 @@ type NodeBuilder struct {
 	withProdConfirmPeriodBlocks bool
 	wasmCacheTag                uint32
 	delayBufferThreshold        uint64
+	useFreezer                  bool
```

Comment:
separate PR: is there actually a good reason to ever set that to false? If not, can remove the config.

Reply:
TestDatabaseConversion fails when freezer is used.